detekt.yml

```diff
@@ -0,0 +1,3 @@
+style:
+  VarCouldBeVal:
+    active: false
```

Comment:
disable it only for test files

Reply:
This warning occurs not only in test files

Reply:
example for non-tests files?

Reply:
In Config file lateinit:
```kotlin
  @Value("\${api.finnhub.token}")
    private lateinit var apiToken: String
```

Reply:
so, u use field injection. 
Let's refactor?